### PR TITLE
Add metastatic history

### PR DESCRIPTION
**Pull request by markkramerus**
New observation added to Patient group indicating whether patient has a history of cancer. The Big Diagram is been updated in the PR.

### CODE REVIEW

File: input/fsh/SD_Patient.fsh [hunk 1]
```diff
@@ -6,3 +6,16 @@ Description: "A patient who has been diagnosed with or is receiving medical trea
...
+
+Profile: HistoryOfMalignantNeoplasm
+Parent: USCoreConditionProblemsHealthConcernsProfile
+Id: mcode-history-of-malignant-neoplasm
+Title: "History of Malignant Neoplasm"
+Description: "Records the existence of a past episode of cancer, for the purpose of long term management and tracking."
+* code from HistoryOfMalignantNeoplasmVS
+
+ValueSet: HistoryOfMalignantNeoplasmVS
+Id: mcode-istory-of-malignant-neoplasm-vs
```
Comment (mlterryMitre1) [inline input/fsh/SD_Patient.fsh:18]: @markkramerus - is this a typo? It has `mcode-istory-of-malignant-neoplasm-vs`

Comment (markkramerus) [reply input/fsh/SD_Patient.fsh]: Fixed

**Comment by saulakravitz** [comment]
I changed the name of the new profile to History of Metastatic Cancer to align with EOM language.
Figure will need to be updated again if this isn't changed.

File: input/includes/markdown-link-references.md [hunk 2]
```diff
@@ -18,6 +18,7 @@
 [GenomicRegionStudied]: StructureDefinition-mcode-genomic-region-studied.html
 [GenomicsReport]: StructureDefinition-mcode-genomics-report.html
 [GenomicVariant]: StructureDefinition-mcode-genomic-variant.html
+[HistoryOfMalignantNeoplasm]: StructureDefinition-mcode-history-of-malignant-neoplasm.html
```
Comment (markkramerus) [inline input/includes/markdown-link-references.md:21]: Wrong name and wrong link

File: input/pagecontent/change_log.md [hunk 3]
```diff
@@ -103,6 +104,10 @@ mCODE is now is explicitly dependent on the [Genomics Reporting IG STU2 (v2.0.0)
 
 The mCODE bundle definition now slices on resource type, rather than profile. Slicing logic was changed because, in some cases, instances could not be assigned unambigously to a slice, causing the FHIR validator to output errors. With this change, the assignment to a slice will always be unambiguous. This change has no effect on the contents or use of the mCODE bundle.
 
+### Addition of Patient History of Malignant Disease
```
Comment (markkramerus) [inline input/pagecontent/change_log.md:107]: Update

**Review by markkramerus — COMMENTED** [review]
i did necessary updates to complete transition to new profile name, value set name, and new SNOMED code.

File: input/includes/markdown-link-references.md (hunk 2 above)
Comment (markkramerus) [reply input/includes/markdown-link-references.md]: Fixed.

**Pull request merged**